app/src/main/java/com/example/vre_app/CreateAccount.kt

```diff
@@ -44,10 +44,10 @@ class CreateAccount : AppCompatActivity() {
                 .isNotEmpty() && confirmPassword.text.toString().isNotEmpty()
         ) {
             //Need a valid Email to work (ex. abc123@gmail.net)
-            if (email.length() in 6..319 && email.text.toString().matches(Regex("[a-zA-Z0-9._]+@[a-z].+[a-z]"))
+            if (email.length() in 6..30 && email.text.toString().matches(Regex("[a-zA-Z0-9._]+@[a-z].+[a-z]"))
```

Comment:
I think the email max length is fine at 319. 
```suggestion
            if (email.length() in 6..319 && email.text.toString().matches(Regex("[a-zA-Z0-9._]+@[a-z].+[a-z]"))
```